src/System.Net.HttpListener/tests/AuthenticationTests.cs

```diff
@@ -30,7 +30,8 @@ public AuthenticationTests()
 
         public void Dispose() => _factory.Dispose();
 
-        [ConditionalTheory(nameof(Helpers) + "." + nameof(Helpers.IsWindowsImplementationAndNotUap))] // Managed implementation connects successfully.
+        [ConditionalTheory(nameof(Helpers) + "." + nameof(Helpers.IsWindowsImplementation))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Not supported on UWP")]
```

Comment:
SkipOnTargetFramework / "Not supported on UWP" is indeed not matching our intent. 
I agree that we should unblock UWP or UWP+AOT testing but we should be careful on the message that we're sending. In the case of all Networking components I think we should use `ActiveIssue` . 

You can either create a new issue or reference one of the issues tracked by Epic #20009.